### ISSUE TITLE
Test / Verify against SLES

### ISSUE THREAD

**Issue by davidnuzik** [k3s-io/k3s#2690]
We would like to expand the K3s OS support matrix and include [SUSE Linux Enterprise Server 15 SP2](https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP2/)

**Comment by davidnuzik** [comment]
Prioritize first testing with 1.20.x, then 1.19, then 1.18. Ideally include 1.17.

**Comment by davidnuzik** [comment]
Bonus: Briefly smoke test on openSUSE Leap (latest).

**Comment by ShylajaDevadiga** [comment]
Following issues were seen with apparmor enabled systems
Deletion of a pod with hostPath backend Persistent Volume hangs on Terminating state.
```
Warning  FailedKillPod  48s (x10 over 2m45s)  kubelet            error killing pod: [failed to "KillContainer" for "task-pv-container" with KillContainerError: "rpc error: code = Unknown desc = failed to stop container \"84d35e9c1cba07a8aa20b2754cb8c5159b0dd192413461560ac761d3a515f4e3\": unknown error after kill: runc did not terminate successfully: container_linux.go:418: signaling init process caused: permission denied\n: unknown", failed to "KillPodSandbox" for "66207d25-e2e9-4c74-aef7-cc74f089dea7" with KillPodSandboxError: "rpc error: code = Unknown desc = failed to stop container \"84d35e9c1cba07a8aa20b2754cb8c5159b0dd192413461560ac761d3a515f4e3\": failed to kill container \"84d35e9c1cba07a8aa20b2754cb8c5159b0dd192413461560ac761d3a515f4e3\": unknown error after kill: runc did not terminate successfully: container_linux.go:418: signaling init process caused: permission denied\n: unknown"]
```
Validated on k3s version 1.19.7+k3s1 and v1.20.2+k3s1 with apparmor disabled at boot.
Cluster config: 2 node, 1 master, 1 agent
Rancher version: v2.5.6-rc3

On v1.19.7+k3s1
Passed following validations:

- Conformance test
- k3s as an imported cluster
- k3s as a local management cluster
- Upgrade of imported cluster from v1.19.7+k3s1 to v1.20.2+k3s1 and run upgrade checks

On v1.20.2+k3s1
Passed following validations:

- Conformance test
- k3s as an imported cluster
- Upgrade of imported cluster from v1.19.7+k3s1 to v1.20.2+k3s1 and run upgrade checks

v1.20.2 is not supported as local management cluster for rancher as of current release.

Keeping issue open to validate with apparmor fix and for now holding off on testing v1.18 and v1.17

**Comment by brandond** [comment]
Apparmor fix is in on K3s 1.18,1.19,1.20 and RKE2 1.19,1.20.

**Comment by rancher-max** [comment]
I have validated the fixes using commits on k3s lines 1.18 (`8c9608b26f07e96a7445f46eeb341a90a33b68a7`), 1.19 (`e34ffdba565067bce8a9da5b79d226f2aad7de4c`), and 1.20 (`1d85a6a30a781ba8644c8f7e3ceec3cc0d21238b`). Standard regression checks and smoketests pass. Will validate conformance and upgrade checks during standard release testing, and then we can close this as validated for the support matrix.

**Comment by rancher-max** [comment]
Release testing on SLES looks good. The new releases in 1.18, 1.19, and 1.20 should be good to use from now going forward.
